Libraries/ActionSheetIOS/NativeActionSheetManager.js

```diff
@@ -17,6 +17,8 @@ import type {ProcessedColorValue} from '../StyleSheet/processColor'; // TODO(mac
 export interface Spec extends TurboModule {
   +getConstants: () => {||};
   +showActionSheetWithOptions: (
+    // TODO(macOS GH#774)
+    // eslint-disable-next-line @react-native/codegen/react-native-modules
```

Comment:
why did we add this comment?

Reply:
We need this as a result of 3a6327a5d9ebfd0de56c37009ab7de1d0e6bdf85 so `yarn lint` doesn't complain.

Reply:
But why doesn't upstream have yarn lint complaints?

Reply:
based on our offline convo, created a new ticket or adding more comments on why this change was needed (share the warning) and we are following the commit 3a6327a change because I don't want our GitHub ticket to be too generic(become a junk drawer)

Reply:
Elaborated on why this divergence exists.